### PR TITLE
Test cases

### PR DESCRIPTION
**Pull request by bhumiijoshi**

### CODE REVIEW

File: myproject/blog/models.py [hunk 1]
```diff
@@ -39,4 +39,7 @@ class Meta:
         db_table = "comments"
     
     def __str__(self):
-        return self.comment
\ No newline at end of file
+        if len(self.comment) > 75:
```
Comment (himanshu-suthar-tagline) [inline myproject/blog/models.py]: Define 75 in the settings.py and use that variable instead.

File: myproject/blog/tests.py [hunk 2]
```diff
@@ -1,3 +1,117 @@
...
+        
+    def test_name_label(self):
+        author=Author.objects.get(id=1)
+        field_label = author._meta.get_field('name').verbose_name
+        self.assertEquals(field_label,'name')
+        
+    def test_biological_label(self):
+        author=Author.objects.get(id=1)
+        field_label = author._meta.get_field('biological_info').verbose_name
+        self.assertEquals(field_label,'biological info')
```
Comment (himanshu-suthar-tagline) [inline myproject/blog/tests.py]: Explain me here what are you testing?

Comment (bhumiijoshi) [reply myproject/blog/tests.py]: All model fields have the correct label.

Comment (himanshu-suthar-tagline) [reply myproject/blog/tests.py]: We don't want to test that. We want to test if the name of the author is correct or any other information related to the author. You are checking if the field exist or not in the database table which is useless and not useful.

**Pull request merged**